pkg/genericapiserver/genericapiserver.go

```diff
@@ -124,8 +123,11 @@ type GenericAPIServer struct {
 	// should be determining the backing storage for the RESTStorage interfaces
 	storageDecorator generic.StorageDecorator
 
-	Mux              *apiserver.PathRecorderMux
+	// HandlerContainer subsumes routes which are published through swagger
 	HandlerContainer *restful.Container
+	// AuxiliaryHandlerContainer subsumes routes which are visible through /, but not through swagger
+	AuxiliaryHandlerContainer *restful.Container
```

Comment:
Something to keep in mind-- we don't want a huge dependency on restful. We may want to switch to a different routing engine and/or swagger assembly method. So I would prefer to see generic apiserver protect this with an interface, so it can be changed easily later (and we can remove the dependency under the hood). Doesn't necessarily need to change in this PR.